domain/src/main/java/org/oppia/domain/profile/ProfileManagementController.kt

```diff
@@ -11,9 +11,11 @@ import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import kotlinx.coroutines.Deferred
 import org.oppia.app.model.Profile
+import org.oppia.app.model.ProfileAvatar
 import org.oppia.app.model.ProfileDatabase
 import org.oppia.app.model.ProfileId
 import org.oppia.data.persistence.PersistentCacheStore
+import org.oppia.domain.R
```

Comment:
Removed unused imports if any and also there are constants which are unused, check that too.